backend/src/main/resources/application.properties

```diff
@@ -13,7 +13,7 @@ gauchocourses.proxy.server=api.ucsb.edu
 gauchocourses.proxy.port=443
 gauchocourses.proxy.scheme=https
 gauchocourses.proxy.api-key-name=ucsb-api-key
-gauchocourses.proxy.api-key-value=
```

Comment:
This shouldn't be necessary, it also causes the build the fail. Are you having trouble connecting to the API without this change?

Reply:
Yeah looks like it's not needed, I was fiddling around with env variables. But now it's having trouble connecting when I try to load more classes or filter by department

Reply:
If you're able, we can do a screenshare during the meeting tonight

Reply:
Sorry, I wasn't able to make the meeting tonight. Yeah for some reason now it's not connecting unless I paste the env var or the API key on that line.

Reply:
What's the name of your env var? If it's "GAUCHOCOURSES_PROXY_API_KEY_VALUE" then Spring _should_ just find and use it

Reply:
Ah it wasn't set to that. I think it's working now for the moment. Thanks!